implementation.md

```diff
@@ -1534,3 +1562,66 @@ Here is an example ad request:
 	“plcmt”: “4” 
 	}
 
+[Top](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/implementation.md#7-implementation-notes-)
+
+## 7.11 - Signaling Podcast Inventory <a name="signalingpodcastinventory"></a>
+
+As the Podcast Industry continues to grow and mature, it is important for the industry to adhere to advertising and content guidelines successfully established in other channels. This guide serves as a tool to consolidate and highlight a clear way to pass the most important signals available today in podcasting through programmatic channels. 
+ 
+While IAB Tech Lab advises that all partners participating in the programmatic buying and selling of podcast inventory should pass all prescribed information below, this is first and foremost a technical guideline built to outline process and does not supersede individual business relationships established by buyers and sellers. It does, however, fully define how data should be sent.
+
+In the tables below, an added column for “Value” is used to define the expected information in the listed field for podcast inventory. This is additional information beyond the description of the field provided formally in OpenRTB and helps to reduce ambiguity about how to use the field to signal podcast inventory.
+
+### 7.11.1 Identity Signals in the Device Object
+
+Programmatic advertising is an operational tool built to simplify the process of buying and selling inventory. To accomplish that simplicity, it is important not only to provide as many data signals as possible for a medium, but also to pass them predictably in the prescribed parameters.
+ 
+Identity signals enable all sides of programmatic advertising to augment data, providing a unique value proposition to their clients. Some channels offer additional ID types, such as cookie IDs, device IDs, and other proprietary solutions as a match point. To supplement for these ID’s, podcast ad servers or SSPs should pass a consistent synthetic ID composed of a hashed IP address and Device User Agent to provide a probabilistic alternative for buyers looking for this type of anchor.
+ 
+IAB Tech Lab recommends passing the following values as identified in the OpenRTB spec:
+
+#### Object: Device
+
+|**Field**  |**Value**      |**Type**|**Description**         |
+|:------|:----------|:------------|:------------------------------------------------------------------|
+|ua |User agent |string |A raw user agent string from the browser. See ua field under device object for more details.|
```

Comment:
I don't see a difference in the usage of `ua`, `sua` for podcasts -- why is this needed? :)